tests/phpunit/tests/functions/wp_nonce_ays.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * @group functions.php
+ * @group query
+ * @covers ::wp_nonce_ays
+ */
+class Tests_Functions_wp_nonce_ays extends WP_UnitTestCase {
+
+	public function test_wp_nonce_ays() {
+		$this->expectException( 'WPDieException' );
+		$this->expectExceptionMessage( 'The link you followed has expired.' );
+		$this->expectExceptionCode( 403 );
+
+		wp_nonce_ays( 'random_string' );
+
+	}
+
+	/**
+	 *
+	 */
+	public function test_wp_nonce_ays_log_out() {
+		$this->expectException( 'WPDieException' );
+		$this->expectExceptionMessageRegExp( '/You are attempting to log out of Test Blog<\/p><p>Do you really want to <a href="http:\/\/example\.org\/wp-login\.php\?action=logout&amp;_wpnonce=.{10}">log out<\/a>\?/m' );
```

Comment:
```suggestion
		$this->expectExceptionMessageMatches( '/You are attempting to log out of Test Blog<\/p><p>Do you really want to <a href="http:\/\/example\.org\/wp-login\.php\?action=logout&amp;_wpnonce=.{10}">log out<\/a>\?/m' );
```

Reply:
The test suite has been upgraded for better PHPUnit cross-version compatibility. You can now use all PHPUnit 9.x assertion and expectation methods and the tests will still work on PHPUnit 5.x - 9.x.
For more details, see https://core.trac.wordpress.org/ticket/46149

Reply:
I saw that the tests were failing and fixed already